electron/main.js

```
@@ -0,0 +1,174 @@
+          result: {
+            rowsAffected,
+            rows,
+            insertId,
+          },
+        };
+
+        results.push(resultInfo);
+      }
+
+      resolve(results);
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)